### ISSUE TITLE
Implement (turn on) emission of token rewards and determine newly minted tokens amount for rewards

### ISSUE THREAD

**Issue by JulissaDantes** [recallnet/ipc#146]
At the end of each voting round, in the block Commit phase, we aim to provide validators a reward for their work, these rewards will be sourced from the newly minted tokens.  

This feature is dependent on and blocked by some feature work in the IPC team.  Currently, no rewards are emitted.  

Once this feature is turned on, rewards will be emitted but only in the native settlement chain token (e.g., ETH or FIL). There will be a separate ticket (#150) to emit rewards in our own custom token.  

This issues is to determine the token amount to be minted in the default token and then implement it.

Update: 
When you pay for storage do you also pay a fee for the transaction that generates or just pay the validator for the storage? Right now, there are zero fees and we only have payment for actual data. **I could see an argument for also charging fees, but this would have to be really well-defined, and probably charged in credits as well to keep costs predictable and stable.** With that in mind, charging fees would be really nice, because it would provide a source of funds for this stuff: https://www.notion.so/Hoku-Proportional-Storage-d7ac3e275e54476196237a1b9a5227ce?pvs=4

Note: **bold** means, needs discussion.
